fract4dgui/gtkfractal.py

```diff
@@ -21,7 +21,6 @@
 
 from . import utils, fourway
 
-
```

Comment:
Two blank lines surrounding class definitions is from PEP 8 (and fract4dgui does now pass a flake8 check!).